aml/org.testeditor.aml.dsl/src/org/testeditor/aml/dsl/scoping/MethodReferenceScopes.xtend

```diff
@@ -32,37 +35,41 @@ class MethodReferenceScopes {
 	@Inject extension IQualifiedNameConverter 
 	
 	/**
-	 * Delegate to appropiate methods depending on the passed reference.
+	 * Provides scoping for MethodReference.operation.
 	 */
-	def IScope getMethodReferenceScope(MethodReference context, EReference reference) {
-		return switch (reference) {
-			case METHOD_REFERENCE__OPERATION: context.scopeFor_MethodReference_operation 
-			// TODO provide scope for type as well - maybe use IImportsConfiguration
-			default: IScope.NULLSCOPE
-		}
+	def IScope scopeFor_MethodReference_operation(MethodReference context) {
+		val fixtureMethods = getFixtureMethods(context.typeReference)
+		return Scopes.scopeFor(fixtureMethods, [simpleName.toQualifiedName], IScope.NULLSCOPE)
 	}
-	
+
 	/**
-	 * Provides scoping for MethodReference.operation.
+	 * Provides scoping for MethodReference.parameters.
 	 */
-	protected def IScope scopeFor_MethodReference_operation(MethodReference context) {
-		val type = context.typeReference?.type
-		if (type instanceof JvmDeclaredType) {
-			val fixtureMethods = type.fixtureMethods
-			return Scopes.scopeFor(fixtureMethods, [simpleName.toQualifiedName], IScope.NULLSCOPE)
-		}
-		return IScope.NULLSCOPE	
+	def IScope scopeFor_MethodReference_parameters(MethodReference context) {
```

Comment:
pretty hard to see in the diff but this is the actual change - added a method for properly scoping the available parameters available in `MethodReference`